src/components/documentBuilder/edit/getElementEditSchemas.tsx

```diff
@@ -21,11 +21,14 @@ import { type DocumentElementType } from "@/components/documentBuilder/documentB
 import React from "react";
 import { VALID_HEADER_TAGS } from "@/components/documentBuilder/allowedElementTypes";
 
-function getClassNameEdit(elementName: string): SchemaFieldProps {
+function getClassNameEdit(
+  elementName: string,
+  { label = "Layout/Style" }: { label?: string } = {}
```

Comment:
Why all of this instead of just:
```suggestion
  label: string
```

Reply:
Future-proofing. I tend to prefer options object args over single variables.

Reply:
In that case could we turn all the params into a single object? Easier for typing and easier to read.